ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6618,28 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+HT2SRC:
```

Comment:
To avoid duplication, I would recommend creating a return type `HT2RWC` to be used with `HT2SWC`.